packages/core/src/generators/move/generator.ts

```diff
@@ -51,6 +56,14 @@ function normalizeOptions(
       /[\\|/]/g,
       '-',
     ),
+    currentNamespace: getNamespaceFromSchema(
+      tree,
+      currentRoot.replace(trimPattern, ''),
+    ),
```

Comment:
This part doesn't work if the namespace has been updated manually or if the project was imported rather than generated. Is there maybe a property inside the csproj we could read for this? There very well may not be, in which case we shouldn't replace things at all. 

I'm really kind of wondering about building a basic AST or tokenizer for .NET, that would make things like this easier to query. Just a thought though, wouldn't be super fun...

Reply:
Good point.   

Renaming the namespaces probably only makes sense in the case the project is using the defaults of the create generators.  
* We could conditionally rename based on whether the project name is consistent with what we expect it to be if not overridden or imported?

AFAIK, filename vs namespaces is only a convention.  

Also, the csproj has `<RootNamespace>` which is optional and only serves as a hint to IDEs when creating new class files.   * I included it in the rename because it shows up in the vbproj.